core-java-modules/core-java/src/main/java/com/baeldung/util/SerializationUtils.java

```diff
@@ -0,0 +1,47 @@
+package com.baeldung.util;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.ObjectInputStream;
+import java.io.ObjectOutputStream;
+import java.io.Serializable;
+import java.lang.reflect.Field;
+import java.lang.reflect.Modifier;
+
+public class SerializationUtils {
+
+	public static <T extends Serializable> byte[] serialize(T obj) throws IOException {
```

Comment:
Indent all Java files with spaces instead of tabs.

Reply:
Looks like i didn't set check-style in my new system :). Done